engines/postgres.py

```diff
@@ -122,6 +124,28 @@ def format_insert_value(self, value, datatype):
                 pass
         return Engine.format_insert_value(self, value, datatype)
 
+    def to_csv(self):
+        tablename = str(Engine.table_name(self))
+        tablefile = str(self.table.name)
+        csvfile_output = os.path.normpath(str(tablefile + '.csv'))
```

Comment:
Are the `str` calls necessary on these three lines? The last line seems to be a string + a string, which should already be a string. It seems likely the values inside the `str`'s on 128 and 129 are strings as well.